candidate-standard/clause_4_terms_and_definitions.adoc

```diff
@@ -1,44 +1,66 @@
+[[terms_and_definitions]]
 == Terms and Definitions
 
 This document uses the terms defined in Sub-clause 5 of https://github.com/opengeospatial/oapi_common/blob/master/19-072.pdf[OGC API-Common Part 1 (OGC 19-072)], which is based on the ISO/IEC Directives, Part 2, Rules for the structure and drafting of International Standards. In particular, the word “shall” (not “must”) is the verb form used to indicate a requirement to be strictly followed to conform to this standard.
 
 For the purposes of this document, the following additional terms and definitions apply.
 
+[[collection-definition]]
+=== Collection
+body of resources that belong or are used together. An aggregate, set, or group of related resources. (<<ogc020-024,OGC 20-024>>)
```

Comment:
Just noting that this definition is way too general and does not describe what an OGC API collection is. We should expect it to change.